### PR TITLE
{lib}[GCCcore/10.3.0] Highway v0.12.2, Brotli v1.0.9, libjxl v0.3.7

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/5e660984b472dd769ef9200e18e98261 for a full test report.

File: easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio
+ comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate
+ but offers more dense compression.
+The specification of the Brotli Compressed Data Format is defined in RFC 7932."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/google/brotli/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['Brotli-%(version)s_pc_link_flags.patch']
```
Comment (verdurin) [inline easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCCcore-10.3.0.eb:16]: @Micket patch is missing?

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 7 out of 8 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/b157ce787c5fca78406e2c642db08d57 for a full test report.

File: easybuild/easyconfigs/l/libjxl/libjxl-0.3.7-GCCcore-10.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+     'source_urls': ['https://skia.googlesource.com/skcms/+archive/'],
+     'extract_cmd': 'tar -C libjxl-%(version)s/third_party/skcms --strip-components=0 -xf %s'},
+
+]
+patches = ['libjxl-%(version)s_openexr-3.0_compat.patch']
+checksums = [
+    '8318a5d1ce82e5683a52512b6f242d2b33a1191e89fc76f7c0ead0a1d60f6c60',  # v0.3.7.tar.gz
+    'f38176fc103fe1f6d23ba6addd5b14e6a54d546dfaa64663306acfe7b6d912ea',  # lodepng-48e5364.tar.gz
+    '1be9a0e16cdd1d2749c8111b862c496100ec959d2efaea7996eee86ef796fddd',  # skcms-6437475.tar.gz
+    'b5bf92b8eb8f41e7cedce1b43ea3fe7a9407d3cc0e29bb2afec652cabe084cee',  # libjxl-0.3.7_openexr-3.0_compat.patch
```
Comment (verdurin) [inline easybuild/easyconfigs/l/libjxl/libjxl-0.3.7-GCCcore-10.3.0.eb:34]: @Micket missing this patch too?

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/1c0ba3bb5fd1668b48ed56715f018975 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/5106305486106821cbbeaa78cbed6703 for a full test report.

File: easybuild/easyconfigs/l/libjxl/libjxl-0.3.7-GCCcore-10.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+     'download_filename': '%s.tar.gz' % local_skcms_version,
+     'source_urls': ['https://skia.googlesource.com/skcms/+archive/'],
+     'extract_cmd': 'tar -C libjxl-%(version)s/third_party/skcms --strip-components=0 -xf %s'},
+
+]
+patches = ['libjxl-%(version)s_openexr-3.0_compat.patch']
+checksums = [
+    '8318a5d1ce82e5683a52512b6f242d2b33a1191e89fc76f7c0ead0a1d60f6c60',  # v0.3.7.tar.gz
+    'f38176fc103fe1f6d23ba6addd5b14e6a54d546dfaa64663306acfe7b6d912ea',  # lodepng-48e5364.tar.gz
+    '1be9a0e16cdd1d2749c8111b862c496100ec959d2efaea7996eee86ef796fddd',  # skcms-6437475.tar.gz
```
Comment (verdurin) [inline easybuild/easyconfigs/l/libjxl/libjxl-0.3.7-GCCcore-10.3.0.eb:33]: @Micket checksum problem?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/libjxl/libjxl-0.3.7-GCCcore-10.3.0.eb (hunk 3 above)
Comment (Micket) [reply easybuild/easyconfigs/l/libjxl/libjxl-0.3.7-GCCcore-10.3.0.eb:33]: Not sure why the first tarball i managed to get a hold off had a different checksum. Fixed

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/f172b201751d600dc5dc834a82fe063f for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/2ba0e9500ac7456d94fb7ad724bd0e97 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/4c15e5aca069b205983debcf38edcb46 for a full test report.

**Comment by verdurin** [comment]
Not sure what's going on here - on one machine I also see `1537a6629e07f4c8bd924766c82237890103f619d1c0eb97d4c29e18b557315f` for the checksum.

**Comment by Micket** [comment]
googlesource's tarball API stores the current filedate. And they don't provide tarballs to download  This sucks

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/9c4a21b85c44052d2ba3443d64ff80c4 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/5089a86bbb164c473db474e1c2a58e7d for a full test report.

**Comment by Micket** [comment]
Shouldn't be any checksum issues anymore; i switched over to lcms

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/2305ce951c01c091be1e12da9ed455da for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/4157e2feca5c6dbb740a93834480c2f6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in total)
node2609.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/f5422b94f9e7a8731516cf3fa6bd9355 for a full test report.

**Pull request merged**